client/src/scenes/challenge/evaluations/components/evaluation.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Evaluation = ({ challenge }) => <p>Evaluation {challenge.id}</p>;
```

Comment:
Could you add some `PropType` validations on this? Wouldn't ESLint yell about this?